### DISCUSSION TITLE
Previous and next movie button

### DISCUSSION THREAD

**Discussion by Dacogetul** [openplayerjs/openplayerjs#402] [Ideas]
Is possible to add previous and next movie button before and after PLAY button ? How can i do that? Thank you.

**Comment by rafa8626** [comment]
@Dacogetul currently the custom buttons are appended based on the position you set them using with the exception of the `right` position. What you can do for now is to create 3 buttons (previous, play and next) and have the `left` controls config empty. Something like:
```javascript
const player = new OpenPlayerJS('id', {
    controls: {
        left: [],
        middle: ['time'],
        right: ['levels', 'captions', 'settings', 'fullscreen'],
    },
});
player.addControl({
    id: 'previous',
    icon: 'https://static.thenounproject.com/png/1372588-200.png',
    title: 'Previous',
    position: 'left',
   click: () => // your operation for PREVIOUS 
});
player.addControl({
    id: 'play',
    icon: 'https://static.thenounproject.com/png/1372588-200.png',
    title: 'Play',
    position: 'left',
   click: () => // your operation for PLAY 
});
player.addControl({
    id: 'next',
    icon: 'https://static.thenounproject.com/png/1372588-200.png',
    title: 'Next',
    position: 'left',
    click: () => // your operation for NEXT 
});
player.init();
```
I hope this helps you.

**Comment by rafa8626** [comment]
@Dacogetul I integrated a new config element when adding a custom control: `index`. If you set the index like this, you will achieve what you are looking for without having to add a custom Play button:
```javascript
const player = new OpenPlayerJS('id', {
    //... your config
});
player.addControl({
    id: 'previous',
    icon: 'https://static.thenounproject.com/png/1372588-200.png',
    title: 'Previous',
    position: 'left',
   index: 0, // This is to prepend it in the `left` position
   click: () => // your operation for PREVIOUS 
});
player.addControl({
    id: 'next',
    icon: 'https://static.thenounproject.com/png/1372588-200.png',
    title: 'Next',
    position: 'left',
    index: 2, // to append it after Play button
    click: () => // your operation for NEXT 
});
player.init();
```
This change hasn't been released yet, but if you are OK with it, I'll release it in the next few days. You can check this in the master branch. Thanks!

**Reply by Dacogetul** [reply]
I tried your version and it didn't work for me. I didn't have any personal settings for the player, so I entered your code directly without mentioning the operations that must be done when "clicking" and it should have shown my icons (pictograms) in the player, but this didn't work. You can see what I did (and possibly what I did wrong) [here](https://codepen.io/Dragos-Brojbeanu/pen/ZEVOGap) .

**Reply by rafa8626** [reply]
@Dacogetul as I said, the change hasn't been released: it's only contained in the master branch. You will need to download the files from the master branch and test this in your local environment. I'm planning to release this by next week if you approve

**Reply by Dacogetul** [reply]
I understand now...thank you very much and sorry for misunderstanding...i wait for next week
